src/frontend/js/components/StepBreadcrumb/index.tsx

```diff
@@ -0,0 +1,101 @@
+// StepBreadcrumb has been created to work in pair with useStepManager hook.
+// Within a step process, it aims to guide the user to know where he/she is in
+// the current process by translating visually the steps manifest.
+import { Fragment } from 'react';
+import { Nullable } from 'types/utils';
+import { Manifest, Step } from 'hooks/useStepManager';
+
+interface Props {
+  manifest: Manifest;
+  step: Nullable<string>;
+}
+
+type FlattenStep = [string, Step];
+
+/**
+ * Retrieve the index of the active step.
+ * When step is null, we know that process is terminated so we have to return
+ * the last step index.
+ * @params step[]
+ * @returns {number} active step index
+ */
+function getActiveStepIndex(steps: FlattenStep[], step: Props['step']) {
+  if (step === null) {
+    return steps.length;
+  }
+  return steps.findIndex((s) => s[0] === step);
```

Comment:
```suggestion
return step === null ? steps.length : steps.findIndex((s) => s[0] === step);
```
a subjective preferred choice, I let it up to you